helpers/lib/cdnify.js

```diff
@@ -30,6 +30,10 @@ module.exports = globals => {
             return path;
         }
 
+        if (['.', '/'].includes(path[0])) {
+            path = path.replace(/^[^a-z\d]*/gi, '');
```

Comment:
Let's just remove `.` and `/` -- make this a strict selection rather than a negation.